### PR TITLE
cd_bootindex: remove cdroms to avoid bootindex conflict

### PR DESCRIPTION
**Pull request by ybduan**
ID: 1821606 
Signed-off-by: ybduan <yduan@redhat.com>

### CODE REVIEW

**Comment by ybduan** [comment]
(1/6) Host_RHEL.m8.u2.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.boot_order_check.bootorder0: PASS (229.76 s)
 (2/6) Host_RHEL.m8.u2.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.boot_order_check.bootorder0.q35: PASS (233.24 s)
 (3/6) Host_RHEL.m8.u2.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.boot_order_check.bootorder1: PASS (229.63 s)
 (4/6) Host_RHEL.m8.u2.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.boot_order_check.bootorder1.q35: PASS (233.23 s)
 (5/6) Host_RHEL.m8.u2.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.boot_order_check.bootorder2: PASS (233.67 s)
 (6/6) Host_RHEL.m8.u2.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.boot_order_check.bootorder2.q35: PASS (238.65 s)

**Comment by ybduan** [comment]
Please help review this PR, thanks! @leidwang

**Comment by leidwang** [comment]
> Please help review this PR, thanks! @leidwang



> Please help review this PR, thanks! @leidwang

LGTM, thanks

**Comment by leidwang** [comment]
(1/6) Host_RHEL.m8.u2.qcow2.virtio_blk.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.boot_order_check.bootorder0: PASS (230.25 s)
 (2/6) Host_RHEL.m8.u2.qcow2.virtio_blk.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.boot_order_check.bootorder0.q35: PASS (233.90 s)
 (3/6) Host_RHEL.m8.u2.qcow2.virtio_blk.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.boot_order_check.bootorder1: PASS (230.39 s)
 (4/6) Host_RHEL.m8.u2.qcow2.virtio_blk.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.boot_order_check.bootorder1.q35: PASS (233.89 s)
 (5/6) Host_RHEL.m8.u2.qcow2.virtio_blk.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.boot_order_check.bootorder2: PASS (234.14 s)
 (6/6) Host_RHEL.m8.u2.qcow2.virtio_blk.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.boot_order_check.bootorder2.q35: PASS (240.06 s)

**Comment by leidwang** [comment]
Acked-by: Leidong Wang leidwang@redhat.com

**Comment by ybduan** [comment]
Please help review this PR, thanks! @luckyh

File: qemu/tests/cfg/boot_order_check.cfg [hunk 1]
```diff
@@ -27,6 +27,7 @@
     force_create_image_stg2 = yes
     remove_image_stg2 = yes
     nic_addr_filter = "%s.*?Bus\s+(\d+),\s+device\s+(\d+),\s+function\s+(\d+)"
+    cdroms = ''
```
Comment (luckyh) [inline qemu/tests/cfg/boot_order_check.cfg:30]: Please leave a comment for this line to describe the issue you met, thanks.

Comment (ybduan) [reply qemu/tests/cfg/boot_order_check.cfg:33]: Updated, please help review again, thanks! @luckyh

**Comment by luckyh** [comment]
Thanks all.

**Pull request merged**